_maps/map_files/BoxStation/BoxStation.dmm

```diff
@@ -37987,22 +37984,23 @@
 /turf/open/floor/plating,
 /area/maintenance/aft)
 "bKM" = (
-/obj/machinery/vending/wallmed{
```

Comment:
These are really hard to replace and are useful as heck, maybe move them to the other side of the room